code/model/DMSDocument.php

```diff
@@ -445,12 +473,30 @@ public function removeAllTags()
 
     /**
      * Returns a link to download this document from the DMS store.
+     * Alternatively a basic javascript alert will be shown should the user not have view permissions. An extension
+     * point for this was also added.
+     *
+     * To extend use the following from within an Extension subclass:
+     *
+     * <code>
+     *
+     * public function updateGetLink($result){
+     *  //Do something here
+     * }
+     * </code>
```

Comment:
Could you please fix the indentation to suit our code style eg: // Do something 
and remove the empty line after `<code>`